ansible/services/nix-komodo/.env

```
@@ -0,0 +1,130 @@
+## so you can pass any additional environment variables to Core / Periphery directly in this file as well.
+
+## Stick to a specific version, or use `latest`
+COMPOSE_KOMODO_IMAGE_TAG=latest
+
+## Note: ðŸš¨ Podman does NOT support local logging driver ðŸš¨. See Podman options here:
+## `https://docs.podman.io/en/v4.6.1/markdown/podman-run.1.html#log-driver-driver`
+COMPOSE_LOGGING_DRIVER=local # Enable log rotation with the local driver.
+
+## DB credentials - Ignored for Sqlite
+DB_USERNAME=admin
+DB_PASSWORD=admin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_USERNAME key

(UnorderedKey)